src/python/tests/test_update_container_definitions.py

```diff
@@ -284,7 +292,7 @@ def test_no_volumes_key(self):
     def test_empty_volumes(self):
         """Updated definition preserves the empty volumes"""
         task_dfn = TestContainerDefinitionsUpdate.task_dfn_empty_volumes
-        ret_val = run(task_dfn, '',  '')
+        ret_val = run(task_dfn, 'container=timer',  '')
```

Comment:
@Xheno Thank you for updating the tests! It looks like they are still failing (see: https://circleci.com/gh/CircleCI-Public/aws-ecs-orb/1979), likely because `image=...` is not supplied. I would recommend running the unit tests locally if you can 🙂

Reply:
Apologies, @lokst I managed to setup pipenv locally and got to run the tests locally. My next push should be good and passing.